nix/hydra-jobs.nix

```diff
@@ -0,0 +1,24 @@
+{ inputs, inputs', pkgs, flake }:
+
+{
+  excludedPaths = [
+    "packages.ghc8107.pre-commit-check"
+    "packages.ghc8107-mingwW64.pre-commit-check"
```

Comment:
Temporarily disable pre-commit-check in CI.
A new PR is needed where fourmolu and possibly editorconfig are run against the entire codebase. After that, we can bring back pre-commit-check in CI

Reply:
Can you add this as a `FIXME comment in the file?